core/src/main/scala/org/bitcoins/core/number/NumberType.scala

```diff
@@ -21,8 +21,8 @@ sealed abstract class Number[T <: Number[T]]
   /** The underlying scala number used to to hold the number */
   protected def underlying: A
 
-  def toInt: Int = toBigInt.bigInteger.intValueExact()
-  def toLong: Long = toBigInt.bigInteger.longValueExact()
+  def toInt: Int = toBigInt.intValue
```

Comment:
Why is this change required? the purpose of using the `Exact()` suffix is so that an error is explicitly thrown in the case of overflows rather than silently "failing" and going back to `Long.MinValue` or `Int.MinValue`

This is especially concerning in a new environment. IIRC the javascript runtime supports different types of numbers, so i would rather these errors be expllicit rather than silently failing.

Reply:
scalajs doesn't have access to these functions for whatever reason, maybe would be best to create our own version in `core/package`